core/build.sbt

```diff
@@ -451,11 +444,7 @@ def cloudflowModule(moduleID: String): Project = {
 
 lazy val commonSettings = Seq(
   organization := "com.lightbend.cloudflow",
-  bintrayOrganization := Some("lightbend"),
-  bintrayRepository := altBintrayRepository.value.getOrElse("cloudflow"),
   headerLicense := Some(HeaderLicense.ALv2("(C) 2016-2019", "Lightbend Inc. <https://www.lightbend.com>")),
```

Comment:
[OT for this PR] is this a left-over from the header plugin? Do we still need this?

Reply:
@maasg I will take a look and fix it in a future PR